store/storetest/link_metadata_store.go

```diff
@@ -250,10 +250,10 @@ func testLinkMetadataStoreTypes(t *testing.T, ss store.Store) {
 		received := result.Data.(*model.LinkMetadata)
 		assert.Nil(t, received.Data)
 
-		result = <-ss.LinkMetadata().Get(metadata.URL, metadata.Timestamp)
-		require.Nil(t, result.Err)
+		linkMetadata, err := ss.LinkMetadata().Get(metadata.URL, metadata.Timestamp)
```

Comment:
probably you can use here the `received, err := ...` and remove the received assignation some lines later.